srcpkgs/adom/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'adom'
 pkgname=adom
 reverts=60_1
```

Comment:
I’m wondering if this line could be removed now, because the predecessor is 3.0.6 and not 60 anymore. Except for some very old installations which haven’t been updated in more than a year. But, however, guess it doesn’t hurt to have it and I’m not sure about the policy in cases like ths. :)

Reply:
I'd be inclined to be safer on this one. That's over my head though whether or not there's a policy.

Reply:
Sure! Was more or less just thinking out loud and wondered if there’s a general rule at all. ;)